dev-packages/ext-scripts/package.json

```diff
@@ -25,8 +25,8 @@
     "ext:watch": "tsc -w -p compile.tsconfig.json",
     "ext:docs": "typedoc --tsconfig compile.tsconfig.json --options ../../configs/typedoc.json",
     "ext:docs:clean": "rimraf docs/api",
-    "ext:test": "nyc mocha --opts ../../configs/mocha.opts \"./src/**/*.*spec.ts\"",
```

Comment:
So from what I understand (sorry I never used jest) it already comes with code coverage reports right? If so would it be possible to remove dependencies to nyc and use this to generate a report? @marcdumais-work This would imply that we also have to change our external sonar instance configuration to use this new report so we still have the code coverage up-to-date am I right?